fe/fe-core/src/main/java/org/apache/doris/analysis/AnalyzeProperties.java

```diff
@@ -42,8 +42,8 @@ public class AnalyzeProperties {
     public static final String PROPERTY_NUM_BUCKETS = "num.buckets";
     public static final String PROPERTY_ANALYSIS_TYPE = "analysis.type";
     public static final String PROPERTY_PERIOD_SECONDS = "period.seconds";
-
     public static final String PROPERTY_FORCE_FULL = "force.full";
+    public static final String PROPERTY_PARTITION_COLUMN_SQL = "partition.column.sql";
```

Comment:
how about "partition.column.from.meta"

Reply:
How about "partition.column.from.sql"? Because by default, it is using meta